code/game/gamemodes/shadowling/shadowling_abilities.dm

```diff
@@ -61,7 +61,7 @@
 	clothes_req = 0
 	range = 5
 	action_icon_state = "veil"
-	var/blacklisted_lights = list(/obj/item/device/flashlight/flare, /obj/item/device/flashlight/slime)
+	var/blacklisted_lights = list(/obj/item/device/flashlight/slime)//Slightly buffed, requiring slimes instead of slimes and flares
```

Comment:
the thing about shadowlings is that they cant put out fire or bio-luminescence for deep lore reasons.